catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/Status.java

```diff
@@ -29,13 +32,16 @@
 
   private final boolean successful;
 
+  private List<String> warnings = new ArrayList<>();
+
   public Status(QueryResponse response, String source, long elapsedTime) {
     elapsed = elapsedTime;
     id = source;
 
     count = response.getResults().size();
     hits = response.getHits();
     successful = isSuccessful(response.getProcessingDetails());
+    warnings = getWarnings(response.getProcessingDetails());
```

Comment:
Is this a potential security risk? Depending on what is placed on `ProcessingDetails`?